### ISSUE TITLE
1.12.2 problem WORLD_THE_END

### ISSUE THREAD

**Issue by LoXR** [ProtocolSupport/ProtocolSupport#884]
Before submitting your issue, please check the following:

* Ensure you are using the latest version of ProtocolSupport from Jenkins: https://build.true-games.org/job/ProtocolSupport/ (`/ps version`)
* Ensure you are using the latest version of Spigot, PaperSpigot or Glowstone as appropriate
* Make sure the problem is actually related to ProtocolSupport - try to reproduce it using only ProtocolSupport
* If possible, you should reproduce your problem with debug enabled (`/ps debug`)
* Check if the issue isn't listed as a won't fix on the project page.
* Please report PE-related issues to https://github.com/ProtocolSupport/ProtocolSupportPocketStuff/issues for now!

Remember, we only support the latest version of ProtocolSupport and the latest server version. Make sure you are not using an old version!

If you fail to check the above, your issue will be closed. Otherwise we're glad to help :-)

Server version: Spigot 1.13
ProtocolSupport Version: Latest Dev
Issue:
So basically 1.13 clients can join the server fine, but for some reason players on lower versions cant join. It had started when someone had joined the end, and it would keep coming up with the error in the picture on lower clients.

Startup: https://pastebin.com/NjTdmLM6
Connect: https://pastebin.com/qcdPY46b
Picture: http://prntscr.com/komt2t

**Comment by Shevchik** [comment]
The latest version is for 1.13.1 server.

**Comment by LoXR** [comment]
Alright, well due to the hosts not having a 1.13.1 build yet, i've had to go back a couple of versions.

**Comment by LoXR** [comment]
Still occurs on latest spigot 1.13.1 build and latest protocolsupport.

**Comment by LoXR** [comment]
Alright re-opening this issue, if I player goes into the world on a lower client version, it will disconnect them and not allow them back in.

**Comment by Speentie8081** [comment]
That's not how github works, you can't reopen a closed issue

**Comment by Shevchik** [comment]
Still need the client+server log and exception, and the exact version.

**Comment by LoXR** [comment]
Client version was 1.13.1, and was the latest protocolsupport.
Dont have any server logs nor client logs, cause it just comes up in the console that the player just disconnected. Its pretty simple to do it again, just put protocolsupport on a 1.13.1 server and go to the end, then the dragon crashes you.

**Comment by Speentie8081** [comment]
Enable debug in server.properties and try to reproduce the issue.

**Comment by Shevchik** [comment]
Try latest build, might be fixed now.

**Comment by LoXR** [comment]
I think its been fixed now, thanks so much for this!
